types/log4js.d.ts

```diff
@@ -270,7 +270,7 @@ export interface Levels {
   FATAL: Level;
   OFF: Level;
   levels: Level[];
-  getLevel(level: string): Level;
+  getLevel(level: string, defaultLevel: Level): Level;
```

Comment:
Might be better with `getLevel(level: Level | string, defaultLevel: Level): Level;`
https://github.com/log4js-node/log4js-node/blob/f8d46a939279c0ab4efc8bb5f0478c4b0949a4cf/lib/levels.js#L22-L31